my/levelshealth.py

```diff
@@ -0,0 +1,41 @@
+'''
+Levels Health data
+'''
+
+REQUIRES = [
+    'git+https://github.com/hpi/levelshealth',
+]
+
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Sequence, Iterable
+
+from .core import Paths, get_files
```

Comment:
we've recently been trying to start switching module code to fully qualified imports (`from my.core import...`) -- the reason is that eventually core will be split as a separate package, and then relative imports in modules won't make much sense
https://github.com/karlicoss/HPI/issues/102#issuecomment-796476003